coprocessor/fhevm-engine/db-migration/migrations/20250708100500_add_transaction_id_for_computations.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE computations
```

Comment:
No index ?
But a sql query has a where clause on it. I am checking with grok since my postgresql skills are on holidays.
It says it need to perform a linear scan but you may know better. Can't the OR clause be completed with something reducing the number of element to check ?